code/modules/mob/living/carbon/superior_animal/human/subtype/voidwolf/voidwolf.dm

```diff
@@ -216,35 +211,68 @@
 	rapid = TRUE
 	rapid_fire_shooting_amount = 3
 	ranged_cooldown = 3
-	projectiletype = /obj/item/projectile/beam/weak/smg
-	drop_items = list(/obj/item/gun/energy/firestorm)
+	projectiletype = /obj/item/projectile/beam/weak/ap/reaver // we do extra damage thanks to the heavy barrel
+	drop_items = list(/obj/item/gun/projectile/automatic/ak47/akl/reaver_modded)
 	limited_ammo = TRUE
 	mag_drop = TRUE
 	rounds_left = 20
-	mag_type = /obj/item/cell/medium/high/depleted
+	mag_type = /obj/item/ammo_magazine/rifle_223/empty
 	mags_left = 3
 
+	casingtype = /obj/item/ammo_casing/beam/ap/spent
+
 	armor = list(melee = 60, bullet = 55, energy = 50, bomb = 75, bio = 100, rad = 25) //Legitmently their armor
 
 /mob/living/carbon/superior_animal/human/voidwolf/elite/New()
 	..()
 	reload_message = "[name] ejects a depleted cell and rapidly reloads a new one!"
 
-
 /mob/living/carbon/superior_animal/human/voidwolf/elite/Initialize()
 
 	allowed_stat_modifiers[/datum/stat_modifier/mob/living/carbon/superior_animal/aggressive/savage] += 10 //10% extra chance to be a glass cannon
 
 	. = ..()
 
+/mob/living/carbon/superior_animal/human/voidwolf/elite/laserak
+	projectiletype = /obj/item/projectile/beam/weak/ap
+	drop_items = list(/obj/item/gun/projectile/automatic/ak47/akl/reaver_modded)
+
+	mag_type = /obj/item/ammo_magazine/rifle_223/empty
+	mags_left = 3
+
+	rapid_fire_shooting_amount = 5 //we're using the burst 5 mode
+	delay_for_rapid_range = 0.22 SECONDS
+
+/obj/item/gun/projectile/automatic/ak47/akl/reaver_modded
+
+	initialized_upgrades = list(/obj/item/gun_upgrade/trigger/dangerzone,
+								/obj/item/tool_upgrade/refinement/laserguide,
+								/obj/item/tool_upgrade/productivity/ergonomic_grip,
+								/obj/item/gun_upgrade/barrel/bore,
```

Comment:
While I very much appreciate the fact the gun has an actual damage mod to justify the 10% increase in damage, this is still a guild-crafted only item. Why would wolves have this?
I would suggest replacing it with a void wolf gauss coil for the sake of a theme, also making them significantly more worth killing

Reply:
the heavy bore is made by the guild but it's not special in any way. It's just a handmade barrel.

Reply:
It's not one of their 'signature' items like the scope that IS infact really, really well made.

Reply:
The heavy barrel is literally the only thing that increases damage at all. I could do some funky stuff with the gaus barrel though.

Maybe later though. Im gonna play divinity.